### PR TITLE
config proguard to build release

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- [#Task 75978: build release](https://edu-redmine.sun-asterisk.vn/issues/75978)
## WHAT
- config proguard to build release
## Evidence (Screenshot or Video)

https://github.com/awesome-academy/HN_Android_2024_Image_Editor_2/assets/163242235/6a0c8f38-23db-4440-a2dd-13a52060aecc

Screenshot 2024-05-17 at 14 10 08
Screenshot 2024-05-17 at 14 09 52
Screenshot 2024-05-17 at 14 09 38
Screenshot 2024-05-17 at 15 26 58
Screenshot 2024-05-17 at 15 26 40

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -35,6 +35,10 @@ android {
             minifyEnabled true
             shrinkResources true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.debug
```
Comment (daolq-2712) [inline app/build.gradle]: signingConfigs.release

File: gradle.properties [hunk 2]
```diff
@@ -20,4 +20,8 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+RELEASE_STORE_FILE=/Users/le.duc.thob/keystore/Untitled/
+RELEASE_STORE_PASSWORD=123456?
+RELEASE_KEY_ALIAS=key0
+RELEASE_KEY_PASSWORD=123456?
```
Comment (daolq-2712) [inline gradle.properties]: bor file này vào .gitignore để không push lên git e nhé!

Comment (ghost) [reply gradle.properties]: vâng ạ

File: .gitignore [hunk 3]
```diff
@@ -25,6 +25,9 @@ build/
 # Local configuration file (sdk path, etc)
 local.properties
 
+
+# Gradle Properties
+gradle.properties
```
Comment (daolq-2712) [inline .gitignore]:
```suggestion
```

File: gradle.properties [hunk 4]
```diff
@@ -20,4 +20,8 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+RELEASE_STORE_FILE=ABC
+RELEASE_STORE_PASSWORD=ABC
+RELEASE_KEY_ALIAS=ABC
+RELEASE_KEY_PASSWORD=ABC?AB
```
Comment (daolq-2712) [inline gradle.properties]:
```suggestion
```

**Review by daolq-2712 — APPROVED** [review]
OK!

**Pull request merged**